docs/src/content/docs/tutorials/split-payments.mdx

```diff
@@ -0,0 +1,296 @@
+      access: [
+        {
+          type: 'quote',
+          actions: ['create', 'read']
+        }
+      ]
+    }
+  }
+)
+```
+
+### 5. Request quotes
+
+Request two quote resources from the customer wallet resource server using the access tokens returned in Step 4. Specify the respective `incomingPayment` URLs returned in Step 3 for each of the quotes.
+
+```ts
+const quote = await client.quote.create(
+  {
+    url: new URL(WALLET_ADDRESS).origin,
+    accessToken: QUOTE_ACCESS_TOKEN
+  },
+  {
+    method: 'ilp',
+    walletAddress: WALLET_ADDRESS,
+    receiver: INCOMING_PAYMENT_URL
+  }
+)
+```
+
+### 5. Request an interactive outgoing payment grant
+
+Request an `outgoingPayment` grant from the customer wallet authorization server.
+
+Add the `limits` object with the following properties:
+
+- `receiveAmount` : the total maximum amount to be received and then split between the merchant's wallet and your platform's wallet.
+
+:::note
+This request requires an interactive grant as the customer will need to consent before an `outgoingPayment` is made against their wallet.
+:::
+
+```ts
+const grant = await client.grant.request(
+  {
+    url: customerWalletAddress.authServer
+  },
+  {
+    access_token: {
+      access: [
+        {
+          identifier: merchantWalletAddress.id,
+          type: 'outgoing-payment',
+          actions: ['read', 'create'],
+          limits: {
+            receiveAmount: {
+              assetCode: receiveAmount.assetCode,
+              assetScale: receiveAmount.assetScale,
+              value: receiveAmount.value
+            }
+          }
+        }
+      ]
+    },
+    interact: {
+      start: ['redirect'],
+      finish: {
+        method: 'redirect',
+        uri: 'http://localhost:3344',
+        nonce: NONCE
+      }
+    }
+  }
+)
+```
+
+### 6. Create outgoing payments
+
+Create two `outgoingPayment` resources on the customer wallet resource server using the access token returned by the authorization server in the grant request in step 5.
+
+<Tabs>
+<TabItem label = 'Merchant'>
+
+Add the following properties:
+
+- `walletAddress` : the URL of the customer wallet address.
+- `quoteId `: the URL of the quote specifying the payment amount to be paid to the merchant.
+
+```ts
+const outgoingPayment = await client.outgoingPayment.create(
+  {
+    url: new URL(WALLET_ADDRESS).origin,
+    accessToken: OUTGOING_PAYMENT_ACCESS_TOKEN
+  },
+  {
+    walletAddress: WALLET_ADDRESS,
+    quoteId: QUOTE_URL
+  }
+)
+```
+
+</TabItem>
+
+<TabItem label ='Platform'>
+Add the following properties:
+
+- `walletAddress` : the URL of the customer wallet address.
+- `quoteId `: the URL of the quote specifying the payment amount to be paid to the platform.
+
+```ts
+const outgoingPayment = await client.outgoingPayment.create(
+  {
+    url: new URL(WALLET_ADDRESS).origin,
+    accessToken: OUTGOING_PAYMENT_ACCESS_TOKEN
+  },
+  {
+    walletAddress: WALLET_ADDRESS,
+    quoteId: QUOTE_URL
```

Comment:
```suggestion
    quoteId: platformQuote.id
```
(as long as we update the getting quote section to set the `platformQuote` and `merchantQuote` varibables)